pkg/loop/cmd/loopinstall/README.md

```diff
@@ -0,0 +1,74 @@
+# loopinstall
+
+A tool for installing Chainlink LOOP plugins from YAML configuration files.
+
+## Installation
+
+To install from source:
+
+```bash
+go install github.com/smartcontractkit/chainlink-common/pkg/loop/loopinstall@latest
+```
+
+Or clone and install locally:
+
+```bash
+git clone https://github.com/smartcontractkit/chainlink-common.git
+cd chainlink-common
+go install ./pkg/loop/loopinstall
+```
+
+## Usage
+
+```bash
+# Run `loopinstall --help` to see the options.
+loopinstall [options] <plugin-config-file> [<plugin-config-file>...]
+```
+
+## Configuration
+
+_(See `plugins.example.yaml` for a complete example configuration.)_
+
+Example configuration file structure:
+
+```yaml
+defaults:
+  goflags: "-ldflags='-s'" # Default Go build flags
```

Comment:
Can we include a comment about why `-s` is used?

Reply:
Added in 2e35011cf7353bd950fb7ac696c41ca142873217